docs/reference/query-dsl/span-multi-term-query.asciidoc

```diff
@@ -37,10 +37,7 @@ GET /_search
 --------------------------------------------------
 // CONSOLE
 
-WARNING: By default `span_multi queries are rewritten to a `span_or` query
-containing **all** the expanded terms. This can be expensive if the number of expanded
-terms is large. To avoid an unbounded expansion you can set the
-<<query-dsl-multi-term-rewrite,rewrite method>> of the multi term query to `top_terms_*`
-rewrite. Or, if you use `span_multi` on `prefix` query only, you can
-activate the <<index-prefix-config,`index_prefixes`>> field option of the `text` field instead. This will
-rewrite any prefix query on the field to a a single term query that matches the indexed prefix.
\ No newline at end of file
+span multi query will hit too many clauses failure if it exceeds the boolean
+query limit (defaults  to `1024`) unless inner multi query uses top_terms_N
+<<query-dsl-multi-term-rewrite,rewrite>> parameter.
+
```

Comment:
Please keep the WARNING here and maybe reword this to:
````
WARNING: `span_multi` queries will hit too many clauses failure if the number of terms that match the query exceeds the boolean query limit (defaults to 1024). To avoid an unbounded expansion you can set the
<<query-dsl-multi-term-rewrite,rewrite method>> of the multi term query to `top_terms_*`
rewrite. Or, if you use `span_multi` on `prefix` query only, you can
activate the <<index-prefix-config,`index_prefixes`>> field option of the `text` field instead. This will
rewrite any prefix query on the field to a a single term query that matches the indexed prefix.
````